src/mds/SessionMap.h

```diff
@@ -393,6 +421,10 @@ class SessionMapStore {
   version_t version;
   ceph::unordered_map<entity_name_t, Session*> session_map;
   PerfCounters *logger;
+
+  // total request load avg
+  DecayRate decay_rate;
+  DecayCounter total_load_avg;
```

Comment:
Let's get rid of `decay_rate`. Add a getter `DecayCounter::get_decay_rate` which can be used instead.

Reply:
I think we can avoid having `DecayCounter::get_decay_rate` altogether -- we can just pass the configured value (fetched from `get_val`) when initializing `DecayCounter` as `DecayRate(double)` is not an explicit constructor.